services/graphql-server/src/graphql/resolvers/platform/content.js

```diff
@@ -447,7 +447,7 @@ module.exports = {
     /**
      * Load primary section of content.
      * If primary section's site matches the current site, return the section.
-     * If not, check for alternative site + section (@todo).
+     * If not, check for alternative site + section.
```

Comment:
The todo comments should actually stay, as they are referring to explicitly allowing a user to set a secondary site/section, not a fallback to a schedule